libraries/hspi_slave.c

```diff
@@ -0,0 +1 @@
+Dummy, should be deleted. 
```

Comment:
I hope you can delete the file in your repository and the deletion is then propagated to master.